README.md

```diff
@@ -33,10 +33,9 @@ data object Amber : Color({ setOf(Red) })
 data object Red : Color({ setOf(Green) })
 ```
 
-<aside>
-Be sure to define your state constructor with functions rather than literal values if you require cycles in your state
-machine. Otherwise, you are likely to encounter null pointer exceptions.
-</aside>
+> ⚠️ Be sure to define your state constructor with _functions_ rather than literal values 
+> if you require cycles in your state machine. Otherwise, you are likely to encounter
+> null pointer exceptions from the Kotlin runtime's inability to define the types.
```

Comment:
```suggestion
> [!CAUTION]
> Be sure to define your state constructor with _functions_ rather than literal values 
> if you require cycles in your state machine. Otherwise, you are likely to encounter
> null pointer exceptions from the Kotlin runtime's inability to define the types.
```

Renders like this:
> [!CAUTION]
> Be sure to define your state constructor with _functions_ rather than literal values 
> if you require cycles in your state machine. Otherwise, you are likely to encounter
> null pointer exceptions from the Kotlin runtime's inability to define the types.